background.js

```diff
@@ -126,13 +127,16 @@ async function moveToBugzilla(data) {
     issue
   });
 
-  const bugzillaRequest = {
+  let bugzillaRequest = {
     api_key: user.bugzilla_key,
     product,
     component,
-    type: bugzillaType(githubData),
+    type: bugType || bugzillaType(githubData),
     version: "unspecified",
-    op_sys: "android",
+    op_sys: opSys,
+    platform: "unspecified",
+    severity,
```

Comment:
What happens if these are undefined?

Reply:
They get the default values as specified by bugzilla - ie, they get the values the bugs would have got before this patch.